pkg/expression/fulltext/analyzer.go

```
@@ -0,0 +1,387 @@
+	tokens = lowerFilter(tokens)
+	return tokens
+}
+
+type parserInfo struct {
+	innodbFtMinTokenSize int
+	innodbFtMaxTokenSize int
+	ngramTokenSize       int
+	stopwords            map[string]struct{}
+}
+
+func parserInfoFromSessionContext(sctx sessionctx.Context) (parserInfo, error) {
```

golangci-lint (2.12.2)

[error] func parserInfoFromSessionContext is unused

(unused)